install-photobooth.sh

```diff
@@ -1075,9 +1075,13 @@ elif [[ \$# -gt 1 ]]; then
     args="\$@"
 fi
 
-sudo systemctl stop go2rtc.service
+if systemctl cat go2rtc.service >/dev/null; then
```

Comment:
Allowed for www-data user? Start and stop was explicitly rights given.

Reply:
Well, depends on the file permissions and the effective umask when the file was created. I don't think anyone has umask 077, so it should work...

Reply:
I think if a user doesn't modify the system it's fine

Reply:
Guess we can and should test this once, capture would fail with error. Can maybe give it a try later

Reply:
I tested it on my pi and it worked. Wasn't Bookworm though..